chat-components/src/main/java/com/pubnub/components/chat/ui/mapper/member/DBMemberMapper.kt

```diff
@@ -11,6 +11,8 @@ class DBMemberMapper : Mapper<DBMemberWithChannels, MemberUi.Data> {
             input.id,
             input.name,
             input.profileUrl,
-            input.member.custom.asMap()?.get("description") as? String?
+            input.member.custom.asMap()?.get("description") as? String?,
```

Comment:
Should we care about `description` when we're passing the complete `custom` object?